readme.md

```diff
@@ -82,7 +85,7 @@ Hello, Node!
 
 * **Question:** Describe what is happening in the above code.
 
-* **Your Answer:**
+* petsFile is a variable that will have the direct path to the pets.json file based on index.js relative path.  So the _dirname = data since data is the folder relative to index.js and pets.json is the file of the directory/name "data".  The join method is combining the directory path, folder name, and file name.
```

Comment:
I'm not sure about this sentence: "So the `__dirname` = data since data is the folder relative to index.js and pets.json"

`__dirname` is the name of the current folder this file is in. You can then join it with other folders to create a full path.